SECURITY.md

```diff
@@ -1,46 +1,32 @@
-- Low: Up to $500 in T tokens.
+- Critical Level: USD $10,000 to USD $25,000
+- High Level: USD $1,000 to USD $10,000
+- Medium Level: USD $1,000
 
-The following attacks are excluded from the Bug Bounty program:
+A great place to begin your research is by working on our testnet. Please see our [documentation](https://docs.threshold.network) to get started. We ask that you please respect network machines and their owners. If you find a vulnerability that you suspect has given you access to a machine against the owner's permission, stop what you're doing and create a report using the immunefi dashboard for researchers.
 
-- Attacks that the reporter has already exploited themselves, leading to damage.
-- Attacks requiring access to leaked keys/credentials.
-- Basic economic governance attacks (e.g. 51% attack).
-- Lack of liquidity.
-- Sybil attacks.
+Rewards are distributed according to the impact of the vulnerability based on the [Immunefi Vulnerability Severity Classification System V2.3](https://immunefi.com/immunefi-vulnerability-severity-classification-system-v2-3/). This is a simplified 4-level scale, with separate scales for websites/apps, smart contracts, and blockchains/DLTs, focusing on the impact of the vulnerability reported.
 
-The following activities are prohibited by this bug bounty program:
+## Reporting a Vulnerability Not Covered by the Bug Bounty Program
 
-- Any testing with mainnet or public testnet contracts; all testing should be done on private testnets.
-- Attempting phishing or other social engineering attacks against our contributors and/or users.
-- Any denial of service attacks.
-- Automated testing of services that generates significant amounts of traffic.
-- Public disclosure of an unpatched vulnerability in an embargoed bounty.
+Please, verify the list of assets in-scope and out-of-scope available as part of the [Threshold Bug Bounty details](https://immunefi.com/bounty/thresholdnetwork/). Additionally, security researchers are encouraged to submit issues outside of the outlined Impacts and Assets in Scope. If you can demonstrate a critical impact on code in production for an asset not in scope, Threshold DAO encourages you to submit your bug report using the “primacy of impact exception” asset in Immunefi.
 
-Rewards are distributed according to the impact of the vulnerability based on the [Immunefi Vulnerability Severity Classification System V2.2](https://immunefi.com/immunefi-vulnerability-severity-classification-system-v2-2/). This is a simplified 5-level scale, with separate scales for websites/apps, smart contracts, and blockchains/DLTs, focusing on the impact of the vulnerability reported.
+Threshold DAO will try to make an initial assessment of a bug's relevance, severity, and exploitability, and communicate this back to the reporter. The Threshold DAO will compensate important findings on a case-by-case basis. We value security researchers and we encourage you to contact us to discuss your findings.
```

Comment:
Should we mention something about a POC here, or since it is defined with Immunefi we don't need to?

Reply:
Same conflicting feeling here. Let's leave it out of here for now, so that Immunefi is the canonical source.

Reply:
I am not opposed to adding something to the topic, but according to immunefi, it needs to be provided or the reports can be closed as not valid. However it did not deter some hackers from submitting low quality proofs. How would you shape it?